tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,2955 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    from utils import SimpleTimer, ResultsLogger, scale_workers
Line #11.  from utils import visualize_data_cuml as visualize_data
</pre><p> </p><p>Import these in a different cell explaining with a comment explaining that you are importing this for benchmarking work.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>